frontend/src/types/ontologyTypes.ts

```diff
@@ -41,6 +41,9 @@ export interface Edge extends UniqueObject {
 export interface GraphEdge extends Edge {
   source: string;
   target: string;
+  sourceToTarget: Edge[];
+  targetToSource: Edge[];
+  flip?: boolean;
```

Comment:
It doesn't seem like this flip attribute is actually used. You're making local flip variables in getRotationAndPosition, but you never put it on the GraphEdge object. I think the flip property can just be removed.

Reply:
JepP! It was ment to be removed for this PR. I am working with a new solution that will reduce rendering time for the flip method.